docs/source/development/CDFs/cdf_introduction.rst

```diff
@@ -0,0 +1,81 @@
+################
+CDF Introduction
+################
+
+
+The Common Data Format, or CDF files, are binary files which require custom libraries to create and manipulate. Goddard Space Flight Center (GSFC) maintains these CDF libraries, and is the shepherd of the file format.
+
+The GSFC describes CDF files as a "self-describing data format for the storage of scalar and multidimensional data in a platform- and discipline-independent way".
+
+IMAP is archiving its science data with the `Space Physics Data Facility (SPDF) <https://spdf.gsfc.nasa.gov/>`_, which requires that data be in the CDF file format.
+
+CDF Ecosystem
+===============
+**CDF** refers to more than just the files themselves, but the entire toolkit surrounding those files.  The official toolkits are available in IDL, C, Fortran and MATLAB.
+
+For python implementations, there is `**cdflib** <https://github.com/MAVENSDC/cdflib>`_, which is a pure-python implementation of the CDF specification. **cdflib** contains the functions "xarray_to_cdf" and "cdf_to_xarray" to convert between xarray Dataset objects in python and the CDF file formats.
+
+.. note::
+   The L0->L2 processing will be using cdflib to generate CDF files on IMAP.  For more information about cdflib and the xarray_to_cdf function, see :ref:`cdflib.xarray_to_cdf`
+
+The other main python library is `**pycdf** <https://spacepy.github.io/pycdf.html>`_, which is a part of **spacepy**.  Pycdf wraps the C libraries, and therefore it requires downloading the C libraries.  Because it is a part of Spacepy, Pycdf requires downloading and installing Spacepy.  Pycdf includes various bells and whistles that help make the files more python friendly, and includes functions for checking for ISTP compliance.
```

Comment:
This shows up as `**pycdf**` in the built docs. I think you might be able to bold it by moving the double star outside the link target. But I don't think you even need to do that and could just let the link target choose how it wants to render since they are usually already italicized or something else.